### DISCUSSION TITLE
moode audio and music assistant

### DISCUSSION THREAD

**Discussion by kuduacz** [music-assistant/support#3261] [Q&A] [answered]
Hi.

I am using MA to stram to Moode audio on my rpi 4b+.
Work well.
Moode audio is detected as dlna renderer in home assistant, and exposed to MA as a HA media player. 
With just one malfunction.
My moode audio dont know what it straming. i mean it is as a radio station playback:
![image](https://github.com/user-attachments/assets/562403a1-141b-4fec-8965-96934997996d)
is there a way to stram to moode audio, where it recognize what is playing now?

in my moode audio i can set it as a airplay renderer, bluetooth, squezlite, plexamp and roonbridge if it helps.

And one more question. 
Is there a way to stream music from my music assistant to web? i try to listen to it when im in work and have to use moode streamer but im preety shure that i can disable moode player to it.

**Comment by OzGav** [comment]
On your first question try adding the DLNA provider to MA and stream to the player directly. Look at the docs for various options to make it work.

No there isn’t a secure way to stream to the web.

**Reply by kuduacz** [reply]
Ty for sugestion.
Successfully adres upnp player direct to ma. Its recogniezed as it should and i can play something to it but its still as radio player![image](https://github.com/user-attachments/assets/1085e9ae-7416-404b-99b7-f1f15b859f5d)

Is there other way to make it work as it should?

Mayen its depend on what i listening on my ma? I using Apple Music provider, and some radio from radio browser. 

Have no idea.
Can You lead me how to make it work?

**Reply by OzGav** [reply]
I don’t understand what you mean. Y

> i can play something to it but its still as radio player

**Reply by OzGav** [reply]
If you are saying it doesn’t have any metadata then try setting it as a squeezelite player or AirPlay.

**Comment by kuduacz** [comment, marked as answer]
I found a solution. in the player settings in music assistant there are options responsible for metadata.

It was previously selected not to send anything. Now I selected profile 2 and it works.

Although it is not perfect either because for some reason the information about what is being played (on moode audio) is refreshed only when changes occur in MA.

So if I go to some playlist and let's say 2 or any other track from it, on moode audio I only see the first position.

**Reply by OzGav** [reply]
It should update as each track changes.

**Comment by kuduacz** [comment]
thanks. by trial and error it works.
there is only one thing that bothers me.
namely the name of the song and album is not displayed correctly. as below.
![image](https://github.com/user-attachments/assets/e395882f-bd25-4d97-bdef-c2a268fdba3c)

But interestingly in the playlist it is ok.
Are these MA settings or directly in the moode audio?

**Reply by OzGav** [reply]
DLNA is poorly standardised. If the device supports Airplay or Squeezelite then use one of those protocols.
